source/views/settings/screens/overview/app-icon.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import * as Icons from '@hawkrives/react-native-alternate-icons'
+import {IconSettingsView} from '../change-icon'
+
+export let AppIconSection = (): JSX.Element => {
```

Comment:
Is `export let` just for consistency here? I think I generally prefer `export const`, but if there's some reason we go this route I have no complaints.

Reply:
I know @hawkrives has voiced a thing or two about preferring let over const across the codebase. Do we want to stick to one over the other?

Reply:
I guess I like `const` at the top level, and `let` within functions?

Reply:
I am okay with that pattern!

Reply:
That's what I was after — I prefer let by default inside of functions, too, but at the top level it makes sense to prevent accidentally changing the value.